osf/features.py

```diff
@@ -8,6 +8,9 @@
 OSF_PREREGISTRATION = 'osf_preregistration'
 OSF_GROUPS = 'osf_groups'
 ENABLE_CHRONOS = 'enable_chronos'
+SLOAN_COI = 'sloan_coi'
```

Comment:
```suggestion
SLOAN_COI = 'waffle:sloan|coi'
```

And so on to match with https://github.com/CenterForOpenScience/osf.io/pull/9022 normalization scheme